documentation/slate/source/includes/apis.md

```diff
@@ -2220,3 +2220,64 @@ no       | depth            | int    | Json response [depth](#depth)
 no       | filter           | string | A [filter](#filter) to refine your request          |
 no       | forbidden_uris[] | id     | If you want to avoid lines, modes, networks, etc.   |
 no       | start_page       | int    | The page number (cf. the [paging section](#paging)) | 0
+
+
+
+<h2 id="Freefloatings-nearby-api">Freefloatings Nearby</h2>
+
+
+``` shell
+#request
+$ curl 'https://api.navitia.io/v1/coverage/<my_coverage>/freefloatings_nearby'
+```
+
+``` shell
+# response, composed by 1 main list: "freefloatings_nearby"
+HTTP/1.1 200 OK
+
+{
+    "free_floatings": [
+        {
+            "public_id": "scooter_12345",
+            "provider_name": "Lime",
+            "id": "scooter_12345",
+            "type": "scooter",
+            "propulsion": "electric",
+            "battery": 85,
+            "distance": 120,
+            "deeplink": "https://lime.com/scooter_12345",
+            "coord": {
+                "lat": "48.8560",
+                "lon": "2.3500"
+            }
+        }
+    ],
+}
+```
+
+The `/freefloatings_nearby` provides access to nearby shared mobility options (such as bikes, scooters, or cars) based on user-provided coordinates.
```

Comment:
The `/freefloatings_nearby` provides > The `/freefloatings_nearby` service provides ?